src/config.js

```diff
@@ -0,0 +1,156 @@
+/*
+ * Module for getting and setting Prebid configuration.
+ *
+ * Prebid previously defined these properties directly on the global object:
+ * pbjs.logging = true;
+ *
+ * Defining and access properties in this way is now deprecated, but these will
+ * continue to work during a deprecation window.
+ */
+const utils = require('./utils');
+
+const DEFAULT_DEBUG = false;
+const DEFAULT_BIDDER_TIMEOUT = 3000;
+const DEFAULT_PUBLISHER_DOMAIN = window.location.origin;
+const DEFAULT_COOKIESYNC_DELAY = 100;
+const DEFAULT_ENABLE_SEND_ALL_BIDS = false;
+
+const ALL_TOPICS = '*';
+
+let listeners = [];
```

Comment:
Chatted about this a little bit offline... For the sake of testability, concurrent auctions, and infinite scroll, I think we should strongly avoid adding global, mutable state to the project. Especially in new modules.

I suggested an alternate pattern, which @matthewlane sounded to like too. I'm pretty sure it'll require a `getConfig` on `$$PREBID_GLOBAL$$`, which kinda sucks... but I think it's a worthwhile tradeoff.

I can describe it here if anyone else cares or wants to hear it.